Gemfile

```diff
@@ -18,6 +18,8 @@ gem 'slim-rails'
 gem 'therubyracer'
 gem 'thin'
 gem 'whenever'
+gem 'sidekiq'
+gem 'sinatra'
```

Comment:
I think we can move this to the `:development` group since it's only needed for the Sidekiq dashboard which we only [set routes for in the `dev` env](https://github.com/fugacious/fugacious/pull/51/files#diff-21497849d8f00507c9c8dcaf6288b136R9).